jujugui/static/gui/src/app/components/error-boundary/test-error-boundary.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+const enzyme = require('enzyme');
+const ErrorBoundary = require('./error-boundary');
+
+class ComponentWithError extends React.Component {
+  constructor(props) {
+    super(props);
+    throw new Error('Error!');
+  }
+  render() {
+    return (
+      <div>
+        <h1>Buggy component</h1>
+      </div>
+    );
+  }
+}
+
+describe('Error boundary', function() {
+
+  const renderComponent = (options = {}) => enzyme.shallow(
+    <ErrorBoundary />
+  );
+
+  it('can render', () => {
+    const wrapper = renderComponent({
+      content: (<span>Children content!</span>)
+    });
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should show error message if available', () => {
+    const renderComponentWithError = (options = {}) => enzyme.shallow(
+      <ErrorBoundary />
+    );
+    const wrapper = renderComponentWithError({
```

Comment:
This can just use the same `renderComponent` above

Reply:
As above, these twp components will receive different content - the reason I've created a new component here is to ensure a freshly mounted component for this test block as they are still within the same test suite. Perhaps this is an unnecessary concern?